website/pages/coil.md

```diff
@@ -0,0 +1,33 @@
+# What is COIL?
+<sup>(Written by [Antar](https://www.smogon.com/forums/members/antar.45129/); converted to Markdown for the PS! website by [Annika](https://www.smogon.com/forums/members/annika.434112/))</sup>
```

Comment:
```suggestion
<small>(Written by [Antar](https://www.smogon.com/forums/members/antar.45129/); converted to Markdown for the PS! website by [Annika](https://www.smogon.com/forums/members/annika.434112/))</small>
```

Fun fact: `<small>` is HTML5 because "fine print" has semantic meaning, but `<big>` isn't because there's no semantic meaning for large text.

Anyway, I'd probably personally prefer to put credits at the bottom of files rather than the top.